osutil/user.go

```diff
@@ -170,3 +170,28 @@ func UidGid(u *user.User) (sys.UserID, sys.GroupID, error) {
 
 	return sys.UserID(uid), sys.GroupID(gid), nil
 }
+
+func UserConfig(usr *user.User, filename string) string {
+	//  phase 0: pretend XDG_CONFIG_HOME is not a thing
```

Comment:
At which phase we are now? Alternatively spread this around the code to indicate what is going on.

Reply:
phase 0 is what we do now

Reply:
Can you make this a little bit more explicit, something like:
```
// This is the rough plan on user configuration transition:
// We are at phase 0 now. The phases are listed below:
```

Reply:
yes, probably. We wouldn't be at phase 0 if you're reading it though ;-) but yes